pkg/kubelet/volumes.go

```diff
@@ -145,7 +145,7 @@ func (kl *Kubelet) getPodVolumes(podUID types.UID) ([]*volumeTuple, error) {
 	for _, volumeKindDir := range volumeKindDirs {
 		volumeKind := volumeKindDir.Name()
 		volumeKindPath := path.Join(podVolDir, volumeKind)
-		volumeNameDirs, err := ioutil.ReadDir(volumeKindPath)
+		volumeNameDirs, err := util.ReadDirNoExit(volumeKindPath)
```

Comment:
nit: it's worth adding a comment here, explaining that it may skip some directories.